test/dart_api/function_test.dart

```diff
@@ -130,4 +129,27 @@ main() {
 
     expect(css, equalsIgnoringWhitespace("a { b: 1; }"));
   });
+
+  test("can compile sass with selector list", () {
+    expect(compileString('''
+a,
+b
+  c: d''', indented: true), equals('''
+a,
+b {
+  c: d;
+}'''));
+  });
```

Comment:
Language test cases should go in [the sass-spec repo](https://github.com/sass/sass-spec). If you make a pull request there, you can link it here by adding "See sass/sass-spec#number" to the PR description. This will cause Travis to run specs against your PR rather than the master branch of sass-spec.